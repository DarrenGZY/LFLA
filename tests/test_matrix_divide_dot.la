function main()
{
    matrix a = [ 1,2;3,4;];
    matrix j =  2.0  /. a;
  	print(j);
}
