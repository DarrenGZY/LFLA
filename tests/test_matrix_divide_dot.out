[[ 2.          1.        ]
 [ 0.66666667  0.5       ]]
